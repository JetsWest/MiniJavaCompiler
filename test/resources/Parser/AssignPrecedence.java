class A {
    public static void main(String[] args) {
        System.out.println(new B().Assign());
    }

}
class B {
    public int Assign() {
        int a;
        int b;
        int c;
        a = 4;
        b = 5;
        c = a + b;
        return c;
    }
}