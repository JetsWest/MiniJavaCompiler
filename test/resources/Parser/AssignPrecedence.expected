Program
  MainClass A
    Print
      new B().Assign()
  Class B
    MethodDecl Assign
      returns int
      int a
      int b
      int c
      a = 4
      b = 5
      c = (a + b)
      Return c